guide/ops/catalog/index.md

```diff
@@ -149,6 +149,12 @@ The following optional catalog metadata is supported:
   and other metadata (such as versions, etc) may not be applied.
 - `brooklyn.libraries`: a list of pointers to OSGi bundles required for the catalog item.
   This can be omitted if blueprints are pure YAML and everything required is included in the classpath and catalog.
+  Each pointer to an OSGi bundle can be a string representing the JAR file url or a map.
+  A map consists of:
+    - `url: http://some.server.or.other/path/my.jar` Resource where JAR file is retrieved . Usually it is with prefix `http://` or ``file://`
+    - `name: myJar-BundleSymbolicName` If not supplied here it will be obtained from <br>`Bundle-SymbolicName` written in `my.jar:META-INF/MANIFEST.MF`
+    - `version: 1.0` If not supplied here it will be obtained from <br>`Bundle-Version` written in `my.jar:META-INF/MANIFEST.MF`
```

Comment:
Don't include this section. There's no good reason why one would need the expanded variant.